src/main/java/net/snowflake/ingest/streaming/internal/ClientBufferParameters.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+import net.snowflake.ingest.utils.ParameterProvider;
+
+/** Channel's buffer relevant parameters that are set at the owning client level. */
+public class ClientBufferParameters {
+
+  private boolean enableParquetInternalBuffering;
+
+  private long maxChunkSizeInBytes;
+
+  private long maxAllowedRowSizeInBytes;
+
+  /**
+   * This is TEST-ONLY constructor
+   *
+   * @param enableParquetInternalBuffering flag whether buffering in internal Parquet buffers is
+   *     enabled
+   * @param maxChunkSizeInBytes maximum chunk size in bytes
+   * @param maxAllowedRowSizeInBytes maximum row size in bytes
+   */
+  public ClientBufferParameters(
```

Comment:
This should either be a static method prefixed with `test_` that calls a private constructor or annotated with `@Test`

Reply:
Good idea! I created the `test_test_createClientBufferParameters` static method.

Reply:
I don't see a `test_createClientBufferParameters` method anywhere?

Reply:
Sorry, I hadn't pushed the commit. It is in https://github.com/snowflakedb/snowflake-ingest-java/blob/f02fa9ec54fb0ae3ffe080abc0f81154204321cd/src/test/java/net/snowflake/ingest/streaming/internal/RowBufferTest.java#L118